modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -112,18 +112,16 @@ def mark_replaced!(superseding_power_of_attorney_request)
         )
     }
 
-    scope :sorted_by, lambda { |sort_column, direction = :asc|
-      case sort_column
+    scope :sorted_by, lambda { |sort_column, direction|
+      direction = direction&.to_s&.downcase
+      normalized_order = %w[asc desc].include?(direction) ? direction : 'asc'
+
+      case sort_column&.to_s
       when 'created_at'
-        order(created_at: direction)
+        order(created_at: normalized_order)
       when 'resolved_at'
-        order_sql = if direction.to_sym == :asc
-                      Arel.sql('ar_power_of_attorney_request_resolutions.created_at ASC NULLS LAST')
-                    else
-                      Arel.sql('ar_power_of_attorney_request_resolutions.created_at DESC NULLS LAST')
-                    end
-
-        includes(:resolution).references(:resolution).order(order_sql)
+        left_outer_joins(:resolution)
+          .order(Arel.sql("resolution.created_at #{normalized_order} NULLS LAST"))
```

Comment:
Still think position of nulls changes relative to ordering (mentioned originally [here](https://github.com/department-of-veterans-affairs/vets-api/pull/21345#discussion_r2019388916))
> should the treatment of nulls in the ordering be related to the ordering being asked for (asc vs desc)?
>
> interpretation-wise, nulls can be thought of "resolution occurs in the future".

Reply:
`     null_treatment = normalized_order == 'asc' ? 'NULLS LAST' : 'NULLS FIRST'`